indico/types/metrics.py

```diff
@@ -0,0 +1,70 @@
+from typing import List
```

Comment:
Similarly I think we may want to either submodule this to model_groups or add model_group in the name of the file for clarity

Reply:
done.